beacon-chain/rpc/prysm/v1alpha1/validator/unblinder.go

```diff
@@ -33,35 +33,39 @@ func unblindBlobsSidecars(block interfaces.SignedBeaconBlock, bundle *enginev1.B
 		return nil, err
 	}
 
+	kzgCommitments := bundle.GetKzgCommitments()
```

Comment:
we didn't do nill checks before on the bundle, do you think it'll be more important now there's an interface?